packages/fast-components-styles-msft/src/call-to-action/index.ts

```diff
@@ -111,12 +116,27 @@ const styles: ComponentStyles<CallToActionClassNameContract, DesignSystem> = {
         ...highContrastForeground,
     },
     callToAction__primary: {
-        "& $callToAction_glyph": {
+        "& $button_contentRegion, $callToAction_glyph": {
             fill: accentForegroundCut,
             ...highContrastSelectedForeground,
         },
-        "&:hover $callToAction_glyph": {
-            ...highContrastHighlightForeground,
+        "&:hover": {
+            "& $callToAction_glyph": {
+                ...highContrastHighlightForeground,
+            },
+        },
+        "a&": {
+            "& $button_contentRegion, & $callToAction_glyph": {
+                ...highContrastLinkForeground,
+            },
+            "&:hover": {
+                [highContrastSelector]: {
+                    "box-shadow": "none !important",
+                },
+                "a& & $button_contentRegion, & $callToAction_glyph": {
```

Comment:
I think this selector can be combined with the above.

Reply:
updated